PinpointKit/PinpointKit/Sources/AnnotationView.swift

```diff
@@ -8,18 +8,16 @@
 
 import UIKit
 
+/// The alpha value used for annotation borders.
+let BorderAlpha: CGFloat = 0.7
```

Comment:
Since this is global, we should name it more specifically or maybe create an `AnnotationViewConstants` enum or struct to store these sorts of things.